itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -1707,6 +1738,50 @@ public void testHdfsNameserviceLazyCopyIncr() throws Throwable {
     }
   }
 
+  @Test
+  public void testHdfsNSLazyCopyIncrExtTbls() throws Throwable {
```

Comment:
both tests can be combined?

Reply:
Bootstrap load is expected to fail in the earlier test, so incremental load failure cannot be tested in the same test.